apps/vaults-v3/components/Filters.tsx

```diff
@@ -9,11 +9,11 @@ import type {ReactElement} from 'react';
 import type {TMultiSelectOptionProps} from '@common/components/MultiSelectDropdown';
 
 type TListHero = {
-	categories: string[];
-	chains: number[];
+	categories: string[] | undefined;
+	chains: number[] | undefined;
```

Comment:
How come explicit `undefined`? Should we do one of these instead:
```suggestion
	categories?: string[];
	chains?: number[];
```
or
```suggestion
	categories: string[] | null;
	chains: number[] | null;
```

Reply:
or even better, not have these as neither `undefined` nor `null` but rather an empty array when there're no categories nor chains. Wdyt?